### PR TITLE
Fix: Close server gracefully on SIGINT and SIGTERM

### PR DESCRIPTION
**Pull request by csansoon**

### CODE REVIEW

File: apps/server/package.json [hunk 1]
```diff
@@ -6,7 +6,9 @@
   "type": "module",
   "scripts": {
     "dev": "vite dev",
-    "build": "svelte-kit sync && vite build",
+    "build:server": "svelte-kit sync && vite build",
+    "build:shutdownPlugin": "mv build/index.js build/server.js && cp plugins/shutdownOnDemand.js build/index.js",
```
Comment (andresgutgon) [inline apps/server/package.json:10]: Why you call it plugins?

File: apps/server/plugins/shutdownOnDemand.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  server.server?.close(() => {
+    process.exit(0);
+  });
+  server.server?.closeIdleConnections();
+  setInterval(() => server.server?.closeIdleConnections(), 1_000);
+  setTimeout(() => server.server?.closeAllConnections(), 20_000);
+}
+
+process.on('SIGINT', shutdownServer);
+process.on('SIGTERM', shutdownServer);
```
Comment (andresgutgon) [inline apps/server/plugins/shutdownOnDemand.js:13]: So this file is not responsible of starting the Svelte server right?

File: apps/server/package.json (hunk 1 above)
Comment (csansoon) [reply apps/server/package.json:10]: Yeah... Should I move it to `scripts`, maybe?

Comment (csansoon) [reply apps/server/package.json]: Moved it to `scripts`

**Review by geclos — APPROVED** [review]
coolio

**Review by andresgutgon — APPROVED** [review]

**Pull request merged**